### ISSUE TITLE
/about and /contact pages

### ISSUE THREAD

**Issue by shinigami-78** [curiositry/mnml-ghost-theme#35]
Hi,

I love your theme for Ghost. I've installed it and gotten most things working except two: I cannot create /about or /contact pages.  I created a static page in Ghost called /about but it doesn't seem to get picked up by the menu like it should. You can check out my blog here to see what I mean: www.war-mango.com.  Try navigating to /about to see that the page exists but it won't show up in the menu.  Could you please help me? Thank you.

**Comment by curiositry** [comment]
@shinigami-78 You’ll need to add the pages to the menu from your blog’s admin panel. I see your about page in the menu already, and your contact page doesn’t seem to be published. Once the pages are up, go to http:// www.war-mango.com/ghost/settings/navigation/ and add entries for your /about and /contact pages, and they’ll show up in the menu.

**Comment by shinigami-78** [comment]
Hey thank you. I did manage to figure it out late, late last night after I
emailed you. Thank you for following up.  If I could ask one more question,
in your mnml blog example your contact page has a form that people can use
to submit messages. Is that something that can be built within the theme or
did you use something like ruby on rails to build it?

Thank you

**Comment by curiositry** [comment]
@shinigami-78 Thanks :)

Ghost doesn’t currently have any provision for contact forms, so I’m using Formspree (https://www.formspree.io/). It seems to be the simplest solution — it's a cinch to set up, and has worked well for me so far.

**Comment by shinigami-78** [comment]
Hey that works awesome! I got it up and running in less than 5 min.  Are
there any other resources you recommend I look at? I am wondering if there
is a way to add any search functionality? Or maybe a list of tags along one
side of the blog to make it easier for people to find topics?

**Comment by curiositry** [comment]
Nice!

There are several options for adding search to a ghost blog, including [Google Custom Search](http://academy.ghost.org/how-to-add-google-custom-search-to-your-ghost-blog/), [GhostHunter](https://github.com/jamalneufeld/ghostHunter), which is powered by Lunr.js, and others. I haven’t found any of the existing solutions satisfactory, so I’m holding out for an official ghost plugin or some more elegant third party solution, mostly because I only have two pages of posts in the first place.

You could add some kind of tag-cloud with the Ghost API, but it isn’t stable yet, so I haven’t added it to any of my themes. What I’ve done over on the Autodidacts is added tag pages for the three main categories to the site menu.

Let me know what you decide on — I’d like to know if there are good options I haven’t run across.

**Comment by shinigami-78** [comment]
I will try that idea to create a tag pages. Thank you!

One last question... is there a way to do TeX in-line instead of being
forced to put everything in a separate line? I am thinking of cases where I
have something simple like S^t_n.

**Comment by curiositry** [comment]
Yes, with some caveats. You can put the .math class on an inline element, such as a span, and KaTeX will parse it. But (as I explain in slightly more detail [here](http://mnml.herokuapp.com/typesetting-mathematics-on-your-ghost-blog-with-tex-mnml-and-katex/)) Ghost parses the content of all non-block-level elements as markdown … and it does this *before* KaTeX parses the equation. This is pain, because if the equation contains markdown formatting characters, it may break the layout. I haven’t found any way around this yet, but let me know if you do.

TL;DR — yes (using .math on a inline element) on a case by case basis; if your equation includes characters that Ghost’s markdown parser will parse into HTML, you’ll probably have to put in a block level element.
